notebooks/book1/03/gauss_infer_2d.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<p>We generally don't use installation directly like this, reason for this is this commands runs everytime irrespective of  package is already installed.  Instead we use something like this,</p><p> </p><p>try:</p><p>&nbsp;&nbsp;import tensorflow as tf</p><p>except:</p><p>&nbsp;&nbsp;%pip install tensorflow</p><p>&nbsp;&nbsp;import tensorflow as tf</p><p> </p><p>from tensorflow import keras</p><p> </p><p>See this <a href="https://github.com/probml/pyprobml/blob/master/notebooks/book1/02/discrete_prob_dist_plot.ipynb" target="_blank">notebook </a>for more information/observations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>